ubjr.c

```diff
@@ -373,7 +374,7 @@ static inline ubjr_array_t priv_ubjr_read_raw_array(ubjr_context_t* ctx)
 		myarray.values = malloc(ls*myarray.size+1);
 		size_t sz = UBJI_TYPE_size[myarray.type];
 
-		if (sz >= 0 && myarray.type != UBJ_STRING && myarray.type != UBJ_HIGH_PRECISION && myarray.type != UBJ_CHAR) //constant size,fastread
+		if (sz >= 0 && myarray.type != UBJ_STRING && myarray.type != UBJ_HIGH_PRECISION && myarray.type != UBJ_CHAR && myarray.type != UBJ_MIXED) //constant size,fastread

```

Comment:
A container can have a size without a type, in which case the type is `UBJ_MIXED`. The array values are `ubjr_dynamic_t` and should not use the fast read.